xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -606,7 +606,7 @@ glusterd_op_txn_begin(rpcsvc_request_t *req, glusterd_op_t op, void *ctx,
     char *volname = NULL;
     uuid_t *txn_id = NULL;
     glusterd_op_info_t txn_op_info = {
-        {0},
+        0,
```

Comment:
I think `GF_OP_STATE_DEFAULT` would be better.

Reply:
OK